content-src/lib/link-menu-options.js

```diff
@@ -161,6 +161,18 @@ export const LinkMenuOptions = {
       data: { url: site.url },
     }),
   }),
+  PinSpocTopSite: (site, index) => ({
+    id: "newtab-menu-pin",
+    icon: "pin",
+    action: ac.AlsoToMain({
+      type: at.TOP_SITES_PIN,
+      data: {
+        site,
+        index,
```

Comment:
may be in a followup bug we send some data that shows its a spoc tile that's pinned, similar to search topsite below